### DISCUSSION TITLE
Widescreen Moguri (UW monitor)

### DISCUSSION THREAD

**Discussion by Crowmeister** [Albeoris/Memoria#501] [Q&A]
When I do a clean install of the game in Steam and install Moguri mod I can only get the widescreen assets when I use a 16:9 aspect-ratio. I have a UW-monitor 3440x1440 so I guess this would make sense.

However, when I afterwards upgrade to the newest Memoria I can not get widescreen assets to work. I tried

- Deleting the memoria ini 
- Disabling widescreen option in Memoria launcher, playing game, stopping game, enabling widescreen option
- Disabling Moguri in Mod Manager, playing game, stopping game, enabling Moguri in Mod Manager
- Borderless, Fullscreen exclusive, window mode, all sorts of aspect-ratios

I might be doing something wrong and it might be because of the aspect-ratio of my monitor, but it DOES work with the (quite old) Memoria Launcher included in Moguri. 

Current settings

![image](https://github.com/Albeoris/Memoria/assets/71778657/0a5aebe0-a243-4e6c-8726-44e4a4591aa8)

Any help would be appreciated.

**Comment by snouz** [comment]
Hi, thanks for the detailed report, this is concerning, it works correctly for me with the same settings, could you share the content of settings.ini, as it's where the resolution is set? I suspect a double value there, which would make the launcher and the "widescreen check" read different values maybe? Just an hypothesis.

**Reply by Crowmeister** [reply]
Well, this is pretty strange. I uninstalled Moguri, uninstalled FF9 from Steam and deleted the whole directory. Then I redid the whole process using [this guide](https://steamcommunity.com/sharedfiles/filedetails/?id=2874210888).

And now it does work! I'm very glad, though not 100% sure which of the steps could have fixed the issue. 

There's a few different steps that MIGHT have helped, like 
- Backupping two asset files and overwriting those after the memoria patcher,  
- Deleting the ini file before installing the new memoria 
- Making a few changes in the freshly generated ini

**Reply by snouz** [reply]
Glad it works now and it's not a fundamental problem, as widescreen is the feature I worked the most on.

Do you still have the old settings.ini somewhere in your bin as I'm still interested to understand what happened, and maybe prevent this problem in the future?

**Reply by Crowmeister** [reply]
This is the memoria ini I used. Seems the key regarding widescreen is only mentioned once.
At first I thought it might be because of aspect ratio of my monitor, but yeah, I didn't run the game with that resolution, only with a 16:9 aspect ratio

Edit: The normal settings.ini looks like this

```
[Settings]
ScreenResolution=2560x1440 | 16:9
ActiveMonitor=0 - AW3423DWF [Primary]
WindowMode=Borderless Fullscreen
[Memoria]
IsDebugMode=False
IsX64=True
CheckUpdates=False
```


[Memoria.ini.txt](https://github.com/Albeoris/Memoria/files/15471969/Memoria.ini.txt)
